### DISCUSSION TITLE
My account got flagged

### DISCUSSION THREAD

**Discussion by abhishekHegde2000** [community/community#83808] [New to GitHub] [answered]
I have raised ticket reinstate my account in GitHub support but have not gotten reply back, its already been 5 days.
can anyone please help me resolve this issue??
when can i expect a reply from the github team??
my ticketid: #2503835

**Comment by queenofcorgis** [comment, marked as answer]
Hi @abhishekHegde2000, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
